homepage/pj1.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+ </head>
+<body>
+    <div id="main">
+        <div class="header">
+            <div class="nav">
+                    <ul>
+                        <section class="nav-left">
+                            <li><a href="#home">Home</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#practical">Practice Areas</a></li>
+                            <li><a href="#News"></a>News</li>
+                            <li><a href="#Contact"></a>Contact</li>                        
+                        </section>
+                        <section class="nav-center">
+                            <li class="logo"><img src="logo.png" alt=""></li>
+                            <li>Alegada</li>
+                        </section>  
+                        <section class="nav-right">
+                            <li>Clients</li>
+                            <li>Recent Cases</li>
+                            <li>Get an Appointment</li>
+                        </section>
+                        <section class="icons">
+                            <li><img src="search.png" alt="Search"></li>
+                            <li><img src="contact.png" alt="Contact"></li>
```

Comment:
I believe this is user icon (not contact) and should be used for login form. Displaying a image without any functionality doesn't make sense?